README.md

```diff
@@ -29,29 +29,75 @@ application, each tutorial is constructed as hybrid theory and programming in an
 interactive environment, removing the gap between theory and implementation.
 
 If you have comments, questions, or would like to contribute to the project
-please feel free to email [me](mailto:dgasmith@gatech.edu).
+please see our [contributor guidelines]().
 
 ### Getting Started
 
-New users can either follow the
-[Getting Started](https://github.com/psi4/psi4numpy/blob/master/Tutorials/01_Psi4NumPy-Basics/1a_Getting-Started.ipynb)
-notebook or follow the less verbose directions below.
+1. Obtain required software
+    1. [Psi4](http://psicode.org/psi4manual/1.1/build_obtaining.html)
+        * Option 1 (easiest): [Download the installer](http://vergil.chemistry.gatech.edu/psicode-download/1.1.html) via the web.  Install according to the instructions [here](http://psicode.org/psi4manual/1.1/conda.html#how-to-install-a-psi4-binary-with-the-psi4conda-installer-download-site).
+        * Option 2 (easy): [Download and install via Conda](http://psicode.org/psi4manual/1.1/conda.html#how-to-install-a-psi4-binary-with-the-psi4conda-installer-command-line)
+        * Option 3 (medium): [Clone & compile from source](https://github.com/psi4/psi4) accoring to the instructions [here](http://psicode.org/psi4manual/1.1/build_planning.html) 
+    2. [Python](https://python.org) 2.7+
+    3. [NumPy](http://www.numpy.org) 1.7.2+
+    4. [Scipy](https://scipy.org) 0.13.0+
```

Comment:
I point them [here](http://psicode.org/psi4manual/1.1/build_obtaining.html#what-is-the-suggested-github-workflow), I could just leave it at that and not reproduce.